packages/web-components/src/tooltip/tooltip.styles.ts

```diff
@@ -0,0 +1,95 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/display.js';
+import {
+  borderRadiusMedium,
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  colorNeutralShadowAmbient,
+  colorNeutralShadowKey,
+  fontFamilyBase,
+  fontSizeBase200,
+  lineHeightBase200,
+  spacingHorizontalXS,
+  spacingVerticalXS,
+} from '../theme/design-tokens.js';
+import { TooltipPositioning } from './tooltip.options.js';
+
+/**
+ * Styles for the tooltip component
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host(:not(:popover-open)) {
+    display: none;
+  }
+
+  :host {
+    --blockOffset: ${spacingVerticalXS};
+    --inlineOffset: ${spacingHorizontalXS};
+    background: ${colorNeutralBackground1};
+    border-radius: ${borderRadiusMedium};
+    border: 1px solid var(--colorTransparentStroke);
+    box-sizing: border-box;
+    color: ${colorNeutralForeground1};
+    display: inline-flex;
+    filter: drop-shadow(0 0 2px ${colorNeutralShadowAmbient}) drop-shadow(0 4px 8px ${colorNeutralShadowKey});
```

Comment:
Is there a preference for `filter` over `box-shadow`?

Reply:
A little future-proofing, but `filter` will work with the arrow better by going around that shape while box-shadow only does the box. Also keeping parity with the React implementation here.